webpack.config.babel.js

```diff
@@ -280,7 +280,7 @@ const baseConfig = {
     __dirname: false,
   },
 
-  devtool: PRODUCTION_BUILD ? 'source-map' : 'eval-cheap-module-source-map',
+  devtool: PRODUCTION_BUILD ? 'source-map' : 'cheap-module-source-map',
```

Comment:
For some reason, the app crashes in development builds when using `eval` after the `active-video-capture` library is loaded. The issue is identical to what's described [here](https://stackoverflow.com/a/69200112). The only downside [appears to be](https://webpack.js.org/configuration/devtool/) that the first build is slower (didn't notice any difference in practice).